cmd/query/app/http_handler_test.go

```diff
@@ -407,6 +407,27 @@ func TestGetTraceBadTimeWindow(t *testing.T) {
 	}
 }
 
+func TestGetTraceWithRawTracesSuccess(t *testing.T) {
+	ts := initializeTestServer(t)
+	ts.spanReader.On("GetTrace", mock.AnythingOfType("*context.valueCtx"), spanstore.GetTraceParameters{
+		TraceID: mockTraceID,
+	}).Return(mockTrace, nil).Once()
+
+	var response structuredResponse
+	err := getJSON(ts.server.URL+`/api/traces/`+mockTraceID.String()+`?raw=true`, &response)
+	require.NoError(t, err)
+	assert.Empty(t, response.Errors)
```

Comment:
the more accurate test would be to use a mock trace that has a known issue that adjusters would fix (e.g. same ID for client/server spans), run the query twice with raw=true|false, and observe the difference in the output.

Reply:
@yurishkuro Agreed. But this PR doesn't actually implement the differentiating logic but rather just passes the flag down. That will happen in https://github.com/jaegertracing/jaeger/pull/6423.